apps/react/src/features/login/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,103 @@
+import { FC, memo, useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { Button, TextField } from '@mui/material';
+import { z, ZodType } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { LoginData } from '@js-camp/core/models/login-data';
+import { useAppDispatch } from '@js-camp/react/store/store';
+import { useNavigate } from 'react-router-dom';
+import { ANIME_PATH } from '@js-camp/react/features/anime/routes';
+import { fetchUser } from '@js-camp/react/store/user/dispatchers';
+import { AuthorizationService } from '@js-camp/react/api/services/authorizationService';
+import { LocalStorageService } from '@js-camp/react/api/services/localStorageService';
+import { Progress } from '@js-camp/react/components/Progress/Progress';
+import { handleServerErrors } from '@js-camp/react/utils/handleServerErrors';
+
+import styles from './LoginForm.module.css';
+
+/** Schema for login form validation. */
+export const LoginSchema: ZodType<LoginData> = z
```

Comment:
Prefer omitting type annotation when it can be inferred. 
```suggestion
export const LoginSchema = z
```